cedar-policy/src/api.rs

```diff
@@ -23,6 +23,12 @@
 )]
 
 mod id;
+#[cfg(feature = "entity-manifest")]
+use cedar_policy_validator::entity_manifest;
+#[cfg(feature = "entity-manifest")]
+pub use cedar_policy_validator::entity_manifest::{
+    AccessTrie, EntityManifest, EntityManifestError, EntityRoot, Fields, RootAccessTrie,
```

Comment:
Nit: we've been trying to keep `pub use`s of error types (like `EntityManifestError`) in the `api/err.rs` file.

Also, for publicly used types, we usually add a "CAUTION..." comment to the type definition as a reminder that it's part of the public interface. Not super important for an experimental feature, but adding it now helps us avoid forgetting it in the future.

Reply:
:+1: